### ISSUE TITLE
Help needed for model reconstruction

### ISSUE THREAD

**Issue by BohanMAT** [juliacamps/FOG#2]
Hi Julia,

I am currently trying to reconstruct your model and I have a few questions that I would like to clarify with you.

1. In your 2018 journal paper, you showed that architecture of your model and used the result from Spectral window stacking as the only input. However, in your code (models.py), there is 1  temporal Convolution layer and using temporal data as input. It seems inconsistent from your journal. May I know the reason why the code is different from the proposed model?

2. Is it possible to share the configuration file? So I can use the optimised setting to reconstruct the model.

Thanks and hope to hear it from you soon.

**Comment by juliacamps** [comment]
Hi,

So, the code in the repo may have a mixture of what I published and a second study that aimed to extend my model with other temporal structures and methods. However, I ran out of time before I could finish this second study, also my supervisor left academia so it was impossible to continue working on it.

I have the intention to clean the repo at some point, but surely not this year, sorry for any confusion caused by it. 

My recommendation would be that you use the repo to understand bits of the paper such as how the data augmentation was implemented and how I fed the data to train the models and that you build your main code from scratch based on the paper especially for the parts linked to Keras since these libraries have changed since I published my work.

Best wishes,

Julià

**Comment by BohanMAT** [comment]
Hi Julià,

Thanks a lot for the prompt reply and suggestions. 

I have another question that I would like to clarify with you. In your check_label function (Under preprocessing_tool.py), it seems that you re-generate the Y labels with a threshold value which results in each SWS window will have one label. Is my interpretation on this correct?

Best Regards,
Bohan

**Comment by juliacamps** [comment]
Yes, that is correct.

**Comment by akamabhoi** [comment]
hi Julia,
I've some doubts regarding the model section of the fog. Can you do this using Conv2d?

**Comment by akamabhoi** [comment]
I'm working on the same project and all the data preprocessing stuff is completed. But I'm stuck in the model implementation section. I'll be grateful if you help me working on the model implementation section.
